fastlane/actions/get_latest_github_release_within_same_major.rb

```diff
@@ -0,0 +1,99 @@
+module Fastlane
+  module Actions
+
+    class GetLatestGithubReleaseWithinSameMajorAction < Action
```

Comment:
You should rename the file to `get_latest_github_release_within_same_major_action.rb`